src/SearchViewProvider.ts

```diff
@@ -152,27 +129,21 @@ export default class SearchViewProvider implements vscode.WebviewViewProvider {
                     <div class="loader js-loader">
                         <div class="loaderBar"></div>
                     </div>
-                    <div class="filters">
-                        <label>
-                            <input type="radio" name="filterBy" value="upcoming" checked class="js-filter" />
-                            current
-                        </label>
-                        <label>
-                            <input type="radio" name="filterBy" value="current" class="js-filter" />
-                            upcoming
-                        </label>
-                    </div>
```

Comment:
Removed the filters here since the SFCCDocs.com website does not support searching for anything more than current docs.